res/values-pt-rBR/strings.xml

```diff
@@ -526,7 +526,7 @@
     <string name="Min_Level">"Nível Mínimo"</string>
     <string name="SAVE">"SALVAR"</string>
     <string name="Player_does_not_meet_the_minimum_level_requirement_to_join_the_clan_">"Este jogador não cumpriu a exigência de nível mínimo para se juntar a este clã:"</string>
-    <string name="Insufficient_Permission">"Permissões Insuficientes"</string>
+    <string name="Insufficient_Permission">"Permissões Insuficientes."</string>
```

Comment:
No dot here. It must follow the blue colour text (those which contain bottom lines). If there isn't any `_` (that means space, punctuations or formatting), then none of these is needed. Please fix it before approval